include/Types.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <blitz/array.h>
+
+namespace blitzdg {
+    using index_type = int;
+    using real_type = double;
+    using vector_type = blitz::Array<double, 1>;
+    using matrix_type = blitz::Array<double, 2>;
+}
```

Comment:
This is legit. I'll add a trello card to get the rest of the source to use the types. and also to move everything into the blitzdg namespace.